### PR TITLE
fix wording in documentation to make it simpler to follow

### PR DESCRIPTION
**Pull request by kopp**
I stumbled over some words in the documentation.
In this PR I tried to 'iron out' those places.

### CODE REVIEW

**Comment by kopp** [comment]
@yo8192 , could you please have a look at this PR?

File: doc/en/fcron.8.sgml [hunk 1]
```diff
@@ -121,18 +121,23 @@ compiled to run in foreground as default.</para>
...
 		    <para>&seealso; option &optserial; in &fcrontab;(5).</para>
 		</listitem>
 	    </varlistentry>
 	    <varlistentry>
 		<term><option>-q</option> <replaceable>num</replaceable></term>
 		<term><option>--queuelen</option> <replaceable>num</replaceable></term>
 		<listitem>
-		    <para>Set to n the number of jobs the serial queue and
-the lavg queue can contain.</para>
+		    <para>Sets the size of the queue that is used to manage serial
```
Comment (thibwk) [inline doc/en/fcron.8.sgml:133]: I would mention the lavg queue on the first line, e.g. (without the docbook formatting):
```
Sets the size of the queue that is used to manage serial jobs and the lavg queue each to <replaceable>num</replaceable>.

This means that:
- at most <replaceable>num</replaceable> serial jobs can be
			in the waiting state.
- at most <replaceable>num</replaceable> jobs can wait until the system average
			load drops below a certain value.
```

WDYT?

File: doc/en/fcrontab.1.sgml [hunk 2]
```diff
@@ -18,7 +18,7 @@ A copy of the license is included in gfdl.sgml.
       </refmeta>
       <refnamediv>
 	 <refname>fcrontab</refname> <refpurpose>manipulate per-user fcrontab
-	    files</refpurpose>
+	    files which adhere to the format defined in fcrontab(5)</refpurpose>
```
Comment (thibwk) [inline doc/en/fcrontab.1.sgml:21]: can you make "fcrontab(5)" a link with:
```
<link linkend="fcrontab.5">&fcrontab;(5)</link>
```

File: doc/en/fcrontab.1.sgml [hunk 3]
```diff
@@ -53,22 +53,26 @@ A copy of the license is included in gfdl.sgml.
 	    linkend="fcron.8">&fcron;(8)</link> daemon. As &fcron; internally
 	    uses a non-human readable format (this is needed because &fcron;
 	    saves more informations than the user gives, for example the time
-	    and date of next execution), the user cannot edit directly his
-	    &fcrontabf; (the one used by &fcron;).</para> <para>When a user
+	    and date of next execution), the user cannot edit his &fcrontabf;
```
Comment (thibwk) [inline doc/en/fcrontab.1.sgml:56]: while we are at it, do you mind replacing "his" by "their" ?

File: doc/en/fcrontab.1.sgml [hunk 4]
```diff
@@ -53,22 +53,26 @@ A copy of the license is included in gfdl.sgml.
...
+		directly (the one used by &fcron;).</para> <para>When a user
 	    installs a &fcrontabf;, the source file is saved in the spool
-	    directory (<filename>&fcrontabsdir;</filename>) to allow future
-	    editions, and a formatted file is generated for the &fcron; daemon,
-	    which is signaled once about ten seconds before the next minute for
-	    all changes made previously. The daemon is not informed of the changes
+	    directory (<filename>&fcrontabsdir;</filename>) to allow it to be
+	    edited, and a formatted file is generated for the &fcron; daemon.
+		This formatted (non human readable) format is sent to the daemon
+		once per minute (about ten seconds before the full minute) with all
```
Comment (thibwk) [inline doc/en/fcrontab.1.sgml:62]: this is now configurable at compile time, with ./configure's `--with-max-fcrontab-reload-delay-seconds`. Do you mind rephrasing as sent with a delay, which by default is once per minute [...] ?

File: doc/en/fcrontab.5.sgml [hunk 5]
```diff
@@ -19,15 +19,16 @@ A copy of the license is included in gfdl.sgml.
     </refmeta>
     <refnamediv>
 	<refname>fcrontab</refname>
-	<refpurpose>tables for driving fcron</refpurpose>
+	<refpurpose>file format to define the schedule that fcron should adhere to</refpurpose>
```
Comment (thibwk) [inline doc/en/fcrontab.5.sgml:22]: How about:
file format to define the jobs that fcron should run, and their schedules.

File: doc/en/readme.sgml [hunk 6]
```diff
@@ -17,24 +17,25 @@ A copy of the license is included in gfdl.sgml.
...
 	<para>In other words, &fcron; does both the job of <application>Vixie
 Cron</application> and <application>anacron</application>, but does even more
 and better :)) ...</para>
 
 	<para>To do so, &fcron; allows you to use the standard mode in which you
-tell it to execute one command at a given date and hour and to make it run a
-command according to its time of execution, which is normally the same as system
-up time. For example:
+tell it to execute one command at a given date and time. Alternatively you can
+instruct &fcron; to execute a command after a certain time of &fcron;'s runtime
```
Comment (thibwk) [inline doc/en/readme.sgml:28]: replace "time" by "duration"

**Review by thibwk — CHANGES_REQUESTED** [review]
Great, thank you!
I left a few comments, can you please take a look?

**Comment by yo8192** [comment]
> Great, thank you! I left a few comments, can you please take a look?

sorry I used the wrong account on the previous post -- just to confirm it was me :)

File: doc/en/fcrontab.1.sgml (hunk 2 above)
Comment (kopp) [reply doc/en/fcrontab.1.sgml:21]: thanks, will do :)

File: doc/en/fcron.8.sgml (hunk 1 above)
Comment (kopp) [reply doc/en/fcron.8.sgml:133]: good idea.  Will not use more listing/indentation in highlighting, as it gets too crowded then,  but just use A/B in text.

File: doc/en/fcrontab.1.sgml (hunk 3 above)
Comment (kopp) [reply doc/en/fcrontab.1.sgml:56]: sure, +1 for diversity :)

File: doc/en/fcrontab.1.sgml (hunk 4 above)
Comment (kopp) [reply doc/en/fcrontab.1.sgml:62]: yes, added slight re-wording around this

File: doc/en/fcrontab.5.sgml (hunk 5 above)
Comment (kopp) [reply doc/en/fcrontab.5.sgml:22]: yes, done

File: doc/en/readme.sgml (hunk 6 above)
Comment (kopp) [reply doc/en/readme.sgml:28]: done

File: doc/en/fcron.8.sgml (hunk 1 above)
Comment (yo8192) [reply doc/en/fcron.8.sgml:133]: Thanks. Looking at it again, with this rewording, I think it should be "sizes of the queues that are used" (plural)?

File: doc/en/fcrontab.1.sgml [hunk 7]
```diff
@@ -53,33 +53,41 @@ A copy of the license is included in gfdl.sgml.
...
+	    directory (<filename>&fcrontabsdir;</filename>) to allow it to be
+	    edited, and a formatted file is generated for the &fcron; daemon.
+		This formatted (non human readable) file with all recent changes
+		included is then sent to the daemon.
+		The daemon is not informed of the changes immediately but after some time
 	    to keep ill disposed users from blocking the daemon by installing
-	    &fcrontabf;s over and over (ie. denial of service attack). We will call
+	    &fcrontabf;s over and over (i.e. denial of service attack).
+		By default, the file is sent once per minute (about ten seconds before
+		the full minute) to the daemon, the parameter
```
Comment (yo8192) [inline doc/en/fcrontab.1.sgml:67]: you should say the "build time parameter" to prevent confusion

File: doc/en/fcrontab.1.sgml [hunk 8]
```diff
@@ -53,33 +53,41 @@ A copy of the license is included in gfdl.sgml.
...
-	    any named file or from standard input if the pseudo-filename "-" is
-	    given, replacing the previous one (if any): each user can have only
+	    any named file or from standard input (if the pseudo-filename "-" is
+	    given), replacing the previous one (if any). Each user can have only
 	    one &fcrontabf;.</para> <para>For instance, root can create a
 	    systemwide fcrontab file, say <filename>/etc/fcrontab</filename>,
 	    and run "<command>fcrontab</command>
 	    <filename>/etc/fcrontab</filename>" to install the new version after
-	    each change of the file. Or (s)he can create a new fcrontab running
+	    each change of the file. Or root can create a new fcrontab running
```
Comment (yo8192) [inline doc/en/fcrontab.1.sgml:90]: not "root" -- did you mean "they" ?

Comment (kopp) [reply doc/en/fcrontab.1.sgml:90]: no, I meant root as this paragraph is about root. In the line befor it says `... root can create ...`, so this is again `root` doing stuff here.
But if you insist, I'll reword to whatever you think is better.
But honestly, it feels like gold plating here -- I think either is fine ;)

File: doc/en/fcrontab.1.sgml (hunk 7 above)
Comment (kopp) [reply doc/en/fcrontab.1.sgml]: done

**Review by yo8192 — APPROVED** [review]

**Comment by yo8192** [comment]
All good, approved and merged. Thank you @kopp !

**Pull request merged**